kukkuu/views.py

```diff
@@ -1,14 +1,22 @@
 import sentry_sdk
 from graphene_django.views import GraphQLView
 
+from kukkuu.exceptions import KukkuuGraphQLError
+
 
 class SentryGraphQLView(GraphQLView):
     def execute_graphql_request(self, request, data, query, *args, **kwargs):
         """Extract any exceptions and send some of them to Sentry"""
         result = super().execute_graphql_request(request, data, query, *args, **kwargs)
         # If 'invalid' is set, it's a bad request
         if result and result.errors and not result.invalid:
-            self._capture_sentry_exceptions(result.errors, query)
+            errors = [
+                e
+                for e in result.errors
+                if not isinstance(e.original_error, KukkuuGraphQLError)
```

Comment:
I think you can't be sure that `error` always has the `original_error` attribute. Can't remember which case was it but I previously implemented like this:
```
        for error in errors:
            try:
                capture(error.original_error)
            except AttributeError:
                capture(error)
```
And I remember ending up catching AttributeError in some cases. But if it's only the cases where `result.invalid` I guess this should do it.

Reply:
Good point, added a check for that.